tests/Integration/goldens/dataproc/src/V1/Gapic/ClusterControllerGapicClient.php

```diff
@@ -136,6 +148,116 @@ private static function getClientDefaults()
         ];
     }
 
+    private static function getClusterNameTemplate()
```

Comment:
Just for my own understanding, why weren't these showing up before? I would have expected to see a lot of deletions from the old properties and methods being removed.

Reply:
This is likely being added because this PR expands the resources considered "in scope" by traversing the entire request message structure for any message or field representing a resource name to ensure that it has a helper generated. See #453

Reply:
That makes sense, but wouldn't we still have _some_ coverage for the resource name helpers which were already being generated?

Reply:
For instance, why aren't these methods being removed? 

https://github.com/googleapis/gapic-generator-php/blob/v2-dev/tests/Integration/goldens/asset/src/V1/Gapic/AssetServiceGapicClient.php#L143

Reply:
Why would they be removed? We aren't taking a breaking change for these until we do the major version rev

Reply:
They're private methods, so they can be removed. I thought we were moving the previous method of generating resource name templates over to using the descriptors. 

So we're only moving to the `ReosurceHelperTrait` for:
 1. V2 clients
 2. V1 clients for previously out-of-scope resource name helpers
Is that correct?

Reply:
I thought we weren't changing V1 Clients at all...literally leaving them the exact same, untouched, no refactoring whatsoever ?

Reply:
OK, I understand. The reason we are doing this is because it requires a GAX version that is incompatible with PHP 5.6.

It's a point to be discussed later, but I do think we should refactor V1 clients to use both `ResourceHelperTrait` and `startApiCall` before we drop support for V1. Otherwise we'll have to maintain both types of calls (using `ResourceNameTrait` vs legacy method, and using `startApiCall` vs legacy method) for the entire compatibility period of about 6 months.

Reply:
This would have to be after we drop support for PHP 5.6